### PR TITLE
Improve ci workflow

### PR DESCRIPTION
**Pull request by AgeManning**
Improves CI workflow

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -220,6 +227,20 @@ type PreviousRlpEncodedValues = Vec<Option<Bytes>>;
...
+pub const IP_ENR_KEY: &str = "ip";
+pub const IP6_ENR_KEY: &str = "ip6";
+pub const TCP_ENR_KEY: &str = "tcp";
+pub const TCP6_ENR_KEY: &str = "tcp6";
+pub const UDP_ENR_KEY: &str = "udp";
+pub const UDP6_ENR_KEY: &str = "udp6";
+#[cfg(feature = "quic")]
+pub const QUIC_ENR_KEY: &str = "quic";
+#[cfg(feature = "quic")]
+pub const QUIC6_ENR_KEY: &str = "quic6";
```
Comment (divagant-martian) [inline src/lib.rs]: you only ever use these as bytes. I think it's easier to go
```rust
pub const QUIC6_ENR_KEY: &[u8] = "quic6";
```
and you can save yourself all those `as_bytes` and similar, and needing to convert from key to string to check the spec reserved keys

**Comment by divagant-martian** [comment]
@AgeManning could you move the workflow improvements to a new pr to keep each change in scope of the pr?

**Comment by AgeManning** [comment]
Yeah. I've got more commits to this locally to. I'm trying trying to integrate it into lighthouse and fixing all the small annoyances.

**Review by divagant-martian — APPROVED** [review]
LGTM!

**Pull request merged**